src/llmcompressor/modifiers/quantization/quantization/mixin.py

```diff
@@ -317,3 +321,38 @@ def _initialize_hooks(self, model: torch.nn.Module) -> Set[RemovableHandle]:
                 hooks.add(self.register_hook(module, calibrate_output_hook, "forward"))
 
         return hooks
+
+    def _untie_if_target_shared_embedding(self, model: torch.nn.Module):
+        """
+        Helper method that checks for shared input/output embedding and unties them
+        if either are targeted by quantization. 
+        """
+        from llmcompressor.transformers.compression.compressed_tensors_utils import (
+            untie_word_embeddings,
+        )
+
+        try:
+            input_embeddings, output_embeddings = (
+                model.get_input_embeddings(),
+                model.get_output_embeddings(),
+            )
+        except NotImplementedError:
+            logger.warning(
```

Comment:
Can we only throw a warning if embeddings are being targeted? That way we don't need to warn the user unless it's relevant to their flow.